### DISCUSSION TITLE
How to add styles for content

### DISCUSSION THREAD

**Discussion by JudgEby** [guocaoyi/create-chrome-ext#30] [Q&A]
Hi! I created project from vanilla-ts template and trying to add button to current open page. The button is added, but I do not know how to add styles to it. I created index.css file in src/content folder, but it doesn't work. Styles from this file are not applied.

**Comment by Ambushfall** [comment]
> Hi! I created project from vanilla-ts template and trying to add button to current open page. The button is added, but I do not know how to add styles to it. I created index.css file in src/content folder, but it doesn't work. Styles from this file are not applied.

Make sure to reference the css file in the manifest for the content script.

Please see guides on chrome extension development 'how to add css to extension script'.

That should nudge you in the right direction
